python/py-argcomplete/Portfile

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-argcomplete
+version             1.9.4
+platforms           darwin
+license             Apache-2
+maintainers         {outlook.com:mohd.akram @mohd-akram} openmaintainer
+
+description         Bash tab completion for argparse
+long_description    ${description}
+
+homepage            https://argcomplete.readthedocs.io
+master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
+distname            ${python.rootname}-${version}
+
+checksums           rmd160  13866fe7923744e726c843da97a807616296d2d2 \
+                    sha256  06c8a54ffaa6bfc9006314498742ec8843601206a3b94212f82657673662ecf1 \
+                    size    47471
+
+python.versions     27 35 36 37
```

Comment:
Since this is a new python port, is there are reason you want to include the old py35 and py36 sub-ports? If not, let's just start with py37 and go from there.

Reply:
Those are the [currently supported versions](https://devguide.python.org/#status-of-python-branches) of Python.

Reply:
Yes, for now. But does anyone actually need them? You are the maintainer, so if you are willing to spend the extra time needed to support older versions, then that is fine. The fewer python versions the lower the maintenance required.

Reply:
Well, I'm convinced. It's all 7s now.